packages/slice-machine/src/features/onboarding/OnboardingGuide/OnboardingGuide.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Card,
+  CardContent,
+  ProgressBar,
+  Text,
+  useMediaQuery,
+} from "@prismicio/editor-ui";
+import clsx from "clsx";
+import { confetti as fireConfetti, ConfettiConfig } from "dom-confetti";
+import { useRef, useState } from "react";
+
+import { OnboardingProgressStepper } from "@/features/onboarding/OnboardingProgressStepper";
+import {
+  OnboardingProvider,
+  useOnboardingContext,
+  useOnOnboardingComplete,
+} from "@/features/onboarding/OnboardingProvider";
+import { useOnboardingExperiment } from "@/features/onboarding/useOnboardingExperiment";
+
+import styles from "./OnboardingGuide.module.css";
+
+const confettiConfig: ConfettiConfig = {
+  colors: ["#8E44EC", "#E8C7FF", "#59B5F8", "#C3EEFE"],
+  elementCount: 200,
+  width: "8px",
+  height: "8px",
+  stagger: 0.2,
+  startVelocity: 35,
+  spread: 90,
+  duration: 3000,
+};
+
+const OnboardingGuideContent = () => {
+  const { steps, completedStepCount, isComplete } = useOnboardingContext();
+  const [isVisible, setVisible] = useState(true);
+  const isMediaQueryVisible = useMediaQuery({ min: "medium" });
+  const confettiCannonRef = useRef<HTMLDivElement>(null);
+
+  useOnOnboardingComplete(() => {
+    const { current: confettiCannon } = confettiCannonRef;
+    if (confettiCannon) fireConfetti(confettiCannon, confettiConfig);
+    setTimeout(() => setVisible(false), confettiConfig.duration);
+  });
+
+  if (!isVisible || !isMediaQueryVisible) return null;
```

Comment:
If we make it invisible via code, why do we need the css?

Reply:
The CSS is only used to the fade out effect. After the transition happens, this boolean will make sure the component is unmounted.

Reply:
I thought introducing `framer-motion`, because it's an awesome tool to achieve great UX with less effort. But its a bit big at 2.19 MB 🫠